### PR TITLE
Python 3.8 support

### PR DESCRIPTION
**Pull request by yan12125**
## Contributor Checklist:

* [x] I have updated the unit tests
* [ ] I have created a file in the `master/buildbot/newsfragments` directory (and read the `README.txt` in that directory)
* [ ] I have updated the appropriate documentation

For now I only make tests pass. Further actions like adding trove classifiers to `setup.py` or a news fragment declaring Python 3.8 support are up to you.

### CODE REVIEW

File: master/buildbot/test/unit/test_changes_bitbucket.py [hunk 1]
```diff
@@ -261,6 +261,9 @@ class TestBitbucketPullrequestPoller(changesource.ChangeSourceMixin,
                                      TestReactorMixin,
                                      unittest.TestCase):
 
+    skip = ('BitbucketPullrequestPoller is broken. '
+            'See https://github.com/buildbot/buildbot/issues/4153')
```
Comment (tardyp) [inline master/buildbot/test/unit/test_changes_bitbucket.py]: I'd rather fix the unit tests and the code. Disabling the test unconditionally is the best way to have bitbucket broken for all pythons

**Comment by yan12125** [comment]
I took another approach that fixes tests on Python 3.8. Anyway, `BitbucketPullrequestPoller` is already broken for _all_ Python versions as described in https://github.com/buildbot/buildbot/issues/4153.

File: .travis.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ language: python
 # Available Python versions:
 # http://about.travis-ci.org/docs/user/ci-environment/#Python-VM-images
 python:
-  - "3.7"
+  - "3.8"
```
Comment (seankelly) [inline .travis.yml]: Should/can 3.8 be added to `.bbtravis.yml`?

**Comment by yan12125** [comment]
> Should/can 3.8 be added to .bbtravis.yml?

Seems Python 3.8 are not installed on workers yet https://buildbot.buildbot.net/#/builders/2/builds/1132

**Comment by tardyp** [comment]
python 3.8 is now available in the docker image

**Comment by yan12125** [comment]
> python 3.8 is now available in the docker image

Thanks! I updated `.bbtravis.yml` to use Python 3.8, too.

**Pull request merged**